CoEpi/en.lproj/Localizable.strings

```diff
@@ -120,7 +120,7 @@
 "UX.cough.titleWet" = "Wet";
 "UX.cough.subtitleWet" = "\nIt feels like there is mucus \nin my throat and/or chest";
 "UX.cough.titleDry" = "Dry";
-"UX.cough.subtitleDry" = "\nMy throat feels 'tickly' or 'scratchy' without mucus";
+"UX.cough.subtitleDry" = "\nMy throat feels 'tickly' or \n'scratchy' without mucus";
```

Comment:
Please ensure that the PRs contain only the changes which are meant to be addressed. This is the only line that should have changed. Use an app like `Sourcetree` or Xcode's git diff to clean up the commits.

Reply:
My apologies. I noticed that in this and another PR after committing. What is the best way to partially revert a commit? should I simply remove the extra files and commit again?

Reply:
files have been removed

Reply:
Thanks. Next time you also can use e.g. [git reset](https://dev.to/konrad_126/git-reset---explain-49bl) in the command line